start.sh

```diff
@@ -240,4 +240,4 @@ startCloudflare
 
 echo
 echo 'Touchbase cluster is launched!'
-echo "Try scaling it up by running: ./start ${CONFIG_FILE} scale"
+echo "Try scaling it up by running: ./start.sh ${CONFIG_FILE} scale"
```

Comment:
Do you need a `-f` before `${CONFIG_FILE}`?

Do you ned `-p` to set the project name?